velox/functions/prestosql/aggregates/tests/MinMaxByAggregationTest.cpp

```diff
@@ -1127,6 +1127,23 @@ TEST_F(MinMaxByComplexTypes, map) {
       {data}, {}, {"min_by(c0, c1)", "max_by(c0, c1)"}, {expected});
 }
 
+TEST_F(MinMaxByComplexTypes, limits) {
```

Comment:
The test name, MinMaxByComplexTypes, suggests that it is used for testing complex types. This test uses integer, which is not a complex type. Consider creating new test class or somehow else refactor to avoid confusion.